testutil/testutil.go

```diff
@@ -20,11 +24,19 @@ import (
 	carv2 "github.com/ipld/go-car/v2"
 	"github.com/ipld/go-car/v2/blockstore"
 	"github.com/ipld/go-ipld-prime"
+	"github.com/libp2p/go-libp2p"
 	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
 	"github.com/multiformats/go-multihash"
 	"github.com/stretchr/testify/require"
 )
 
+func NewID(t testing.TB) peer.ID {
+	lp2p, _ := libp2p.New()
+	defer lp2p.Close()
```

Comment:
If all we want from a libp2p host is an ID, then we can generate random a id like [this](https://github.com/filecoin-project/storetheindex/blob/357c23af1b6929885a2ddf7329a1537e2449b6c5/config/init.go#L46). 

Cid V0 can also be a valid peer ID I think.

Reply:
thanks it worked